index.html

```diff
@@ -90,55 +90,49 @@ <h3>Canonical HTTP APIs</h3>
 
     <section id="getting-started">
       <h2>Getting Started</h2>
-      <div id="cargo-install">
-        <p>Install via Cargo, initialize, and run the daemon</p>
-        <pre data-enlighter-language="shell">
-$ cargo install ipfs-http
-$ ipfs-http init --bits 4096 --profile test
-$ ipfs-http daemon &amp;
-...
-API listening on /ip4/127.0.0.1/tcp/33801
-daemon is running
-
-$ curl 127.0.0.1:33801/api/v0/id | jq .ID
-"QmRJUsPnZH5Pn3dBRDDiEV1JBjLB1KTeFhspYwFVk8czS1"
-        </pre>
-      </div>
-      <div>
-        <p>...or use it in your Rust code</p>
-        <pre data-enlighter-language="rust">
-use ipfs::{IpfsOptions, IpfsPath, TestTypes, UninitializedIpfs};
-use std::convert::TryFrom;
+      <p>
+        First, add <code>ipfs</code> and <code>tokio</code>
+        to your <strong>Cargo.toml</strong> file:
+      </p>
+      <pre data-enlighter-language="toml">
+[dependencies]
+futures = { default-features = false, version = "0.3.5", features = ["alloc", "std"] }
+ipfs = "0.2.0"
```

Comment:
```suggestion
ipfs = "0.2.1"
```

Not sure if "0.2" would suffice. I know "0.2.0" will not work :)